community_images/common/tests/common_commands.sh

```diff
@@ -1,32 +1,30 @@
-#!/bin/bash
+#!/bin/sh
 
 set -x
 set -e
 
 # Add common commands here which should be present in all hardened images
 
-function run_commands()
+run_commands()
 {
-    local cmd="$1"
```

Comment:
Why this change?

Reply:
need to use POSIX shell instead of bash for traefik image

Reply:
does it break anything else?